improver_tests/utilities/test_CalculateForecastValueFromClimateAnomaly.py

```diff
@@ -0,0 +1,458 @@
+    return cube
+
+
+@pytest.fixture
+def standardized_anomaly_cube_multiple_time_points(
+    validity_time, forecast_reference_time, mean_cube
+):
+    """
+    Fixture for creating a standardized gridded anomaly cube with multiple
+    time points.
+    """
+    cubes = CubeList()
+    for time, value in zip(validity_time["anomaly_multiple"], [0.5, 0.0, -0.5]):
+        data = np.full((1, 1), value, dtype=np.float32)
+        cubes.append(
+            set_up_variable_cube(
+                data=data, time=time, frt=forecast_reference_time, units="1"
+            )
+        )
+    cube = cubes.merge_cube()
+    cube.long_name = "air_temperature_standardized_anomaly"
+    add_reference_epoch_metadata(cube, mean_cube)
+    return cube
+
+
+# Site cube fixtures
+# Site anomaly cubes with mutliple time points are not created for testing as the
+# plugin's functionality is not expected to change, and so is implicitly tested by the
+# gridded anomaly cubes with multiple time points.
+
+
+@pytest.fixture
+def site_cubes(validity_time, time_bounds, forecast_reference_time, mean_cube):
+    """Fixture for creating site cubes."""
+    # Unstandardized anomaly site cube
+    data = np.array([7.0], dtype=np.float32)
+    unstandardized_anomaly_site_cube = set_up_spot_variable_cube(
+        data=data,
+        time=validity_time["anomaly_basic"],
+        frt=forecast_reference_time,
+        units="K",
+    )
+    unstandardized_anomaly_site_cube.standard_name = "air_temperature_anomaly"
+    add_reference_epoch_metadata(unstandardized_anomaly_site_cube, mean_cube)
+
+    # Standardized anomaly site cube
+    data = np.array([1.75], dtype=np.float32)
+    standardized_anomaly_site_cube = set_up_spot_variable_cube(
+        data=data,
+        time=validity_time["anomaly_basic"],
+        frt=forecast_reference_time,
+        units="1",
+    )
+    standardized_anomaly_site_cube.long_name = "air_temperature_standardized_anomaly"
+    add_reference_epoch_metadata(standardized_anomaly_site_cube, mean_cube)
+
+    # Mean site cube
+    data = np.array([298], dtype=np.float32)
+    mean_site_cube = set_up_spot_variable_cube(
+        data=data,
+        time=validity_time["mean_and_std"],
+        time_bounds=time_bounds["mean_and_std"],
+        units="K",
+    )
+    cell_method = iris.coords.CellMethod(method="mean", coords="time")
+    mean_site_cube.add_cell_method(cell_method)
+
+    # Standard deviation site cube
+    data = np.array([4], dtype=np.float32)
+    std_site_cube = set_up_spot_variable_cube(
+        data=data,
+        time=validity_time["mean_and_std"],
+        time_bounds=time_bounds["mean_and_std"],
+        units="1",
+    )
+    cell_method = iris.coords.CellMethod(method="std", coords="time")
+    std_site_cube.add_cell_method(cell_method)
+
+    return (
+        unstandardized_anomaly_site_cube,
+        standardized_anomaly_site_cube,
+        mean_site_cube,
+        std_site_cube,
+    )
+
+
```

Comment:
Many of these tests need modifying due to my suggestions for modifications to the plugin or the input data. We can talk about what needs changing in more detail if required.